test/java/seedu/addressbook/data/SortTest.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data;
+
+import org.junit.Test;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.util.TestUtil;
```

Comment:
Imports from java and third parties should come before `seedu` imports, separated by blank lines :)